Seaworthy/src/app/account/account.component.ts

```diff
@@ -1,15 +1,97 @@
 import { Component, OnInit } from '@angular/core';
+// import { MdInputModule } from '@angular/material';
+import { FormControl, Validators } from '@angular/forms';
+// import { MdAutocompleteModule } from '@angular/material';
+
+import 'rxjs/add/operator/startWith';
+import 'rxjs/add/operator/map';
+
+const EMAIL_REGEX = /^[a-zA-Z0-9.!#$%&’*+/=?^_`{|}~-]+@[a-zA-Z0-9-]+(?:\.[a-zA-Z0-9-]+)*$/;
 
 @Component({
   selector: 'app-account',
   templateUrl: './account.component.html',
-  styleUrls: ['./account.component.css']
+  styleUrls: ['./account.component.css'],
 })
 export class AccountComponent implements OnInit {
 
-  constructor() { }
-
-  ngOnInit() {
+  submitForm() {
+    alert("YO");
   }
 
-}
+  emailFormControl = new FormControl('', [
+    Validators.required,
+    Validators.pattern(EMAIL_REGEX)]);
+
+    stateCtrl: FormControl;
+    filteredStates: any;
+
+    states = [
```

Comment:
You can put this array data in another file and import it, is a better practice.